src/main/java/com/booster/server/NoteController.java

```diff
@@ -41,4 +41,10 @@ List<NoteDto> list(@RequestParam Integer size) {
         return noteService.batchOfSize(size);
     }
 
+    @DeleteMapping("/{id}")
+    @ResponseStatus(HttpStatus.NO_CONTENT)
```

Comment:
Add the same response status for the update endpoint.

Reply:
Done